core/templates/pages/admin-page/activities-tab/admin-dev-mode-activities-tab.component.ts

```diff
@@ -0,0 +1,213 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for the activities tab in the admin panel when Oppia
+ * is in developer mode.
+ */
+
+import { Component, Output, OnInit, EventEmitter } from '@angular/core';
+
+import { AdminDataService } from 'pages/admin-page/services/admin-data.service';
+import { AdminTaskManagerService } from 'pages/admin-page/services/admin-task-manager.service';
+import { WindowRef } from 'services/contextual/window-ref.service';
+import { downgradeComponent } from '@angular/upgrade/static';
+import { AdminBackendApiService } from 'domain/admin/admin-backend-api.service';
+
+@Component({
+  selector: 'oppia-admin-dev-mode-activities-tab',
+  templateUrl: './admin-dev-mode-activities-tab.component.html',
+})
+export class OppiaAdminDevModeActivitiesTabComponent implements OnInit {
```

Comment:
We don't have to add Oppia prefix in the class name. [Ditto elsewhere]

@srijanreddy98, Do we have any guidelines/doc on how to create new components?

Reply:
Made the changes

Reply:
@DubeySandeep we don't have that yet... I will create one. Thanks for flagging this.